### ISSUE TITLE
Files removed from dev directory are not removed from parent directory

### ISSUE THREAD

**Issue by benoitchantre** [wprig/wprig#101]
## Issue Overview
When you remove a file from the `dev` directory, it is not removed from the parent directory by the build process. This can lead to load an unwanted template.

## Expected Behavior
Files removed from the `dev` directory should be removed automatically.

**Comment by mor10** [comment]
This is correct. There is currently no feature for removing removed files. Something to think about for v1.1. I don't have an immediate answer to how the logic would work for this, but it's possible @ataylorme does.

**Comment by ataylorme** [comment]
I don’t know of a way off the top of my head. Whenever I’ve worked with a build step before the production files go to a sub directory as well which can be deleted and recreated when building for production. 

We could revamp the bundle script to create a production/build directory. The production directory could then be used to create the zip file bundle currently creates.

Thoughts @mor10?

**Comment by felixarntz** [comment]
This will be no longer relevant after #133.
